test/api/v3/integration/groups/POST-groups_groupId_join.test.js

```diff
@@ -130,6 +155,14 @@ describe('POST /group/:groupId/join', () => {
         await expect(invitedUser.get('/user')).to.eventually.not.have.deep.property('invitations.party.id');
       });
 
+      it('increments memberCount when joining party', async () => {
+        let oldMemberCount = party.memberCount;
```

Comment:
I think you also need a separate test for testing a member is now in that party.